utilities/install-collector.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2022
+
+# This script installs community maintained collectors from netdata/community github repo
+
+NETDATA_DIR="/etc/netdata"
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root."
+  exit
+fi
+
+if [ ! -d $NETDATA_DIR ]; then
+    printf "\nPlease update NETDATA_DIR in %s with the Netdata user configuration directory as mentioned in https://learn.netdata.cloud/docs/agent/collectors/plugins.d#environment-variables\n" "$0"
+    exit 1
+fi
+
+printf "\nYou are about to install a collector that is NOT developed or maintained by Netdata. Would you like to proceed? [y/n]"
+read -r answer
+
+if [ "$answer" = "y" ] || [ "$answer" = "Y" ]; then
+    printf "\nInstalling collector from community repo..."
+else
+    printf "\n\nExiting...\n\n"
+    exit 1
+fi
+
+input=$1
+collector=${input#*/}
+suffix=".plugin/$collector"
+plugin=${input%"$suffix"}
+
+if [ "$plugin" == "python.d" ]; then
+    file_ext=".py"
+elif [ "$plugin" == "charts.d" ]; then
+    file_ext=".sh"
+elif [ "$plugin" == "go.d" ]; then
+    file_ext=".go"
+elif [ "$plugin" == "node.d" ]; then
+    file_ext=".js"
+fi
+
+charts="https://raw.githubusercontent.com/netdata/community/main/collectors/$1/$collector.chart$file_ext"
+config="https://raw.githubusercontent.com/netdata/community/main/collectors/$1/$collector.conf"
+executable="/usr/libexec/netdata/$plugin/"
+collector_conf="/etc/netdata/$plugin/"
+conf="/etc/netdata/$plugin.conf"
+enabled="\"$collector: yes\""
+filecheck="$collector_conf/$collector.conf"
+
+if [ -f "$filecheck" ]; then
+  printf "\n\nA Netdata collector by this name already exists on this system. Exiting...\n\n"
+  exit 1
+fi
+
+sudo wget "$charts" -P "$executable"
```

Comment:
Here  you are assuming that `wget` is always installed, unfortunately this is not true. Users can have only `curl` on their environment, please, take a look at next link

https://github.com/netdata/netdata/blob/b8d50ecc953f7962587aaae1a793827faf4ce992/packaging/installer/functions.sh#L109-L129

We should have something like this for this script.

Reply:
Added